src/sixel.h

```diff
@@ -173,6 +177,8 @@ namespace MR {
           }
           // replace last character from $ (carriage return) to '-' (newline):
           out.back() = '-';
+          if (need_newline_after_sixel)
+            out += VT::move_cursor (VT::Down,1) + VT::CarriageReturn;
```

Comment:
this line breaks sixels in iTerm2, I get text to the terminal instead of the image

Reply:
Sorry, my mistake... But it looks like you've already figured it out?